test/test_image.py

```diff
@@ -158,6 +159,45 @@ def test_encode_png(self):
                 RuntimeError, "The number of channels should be 1 or 3, got: 5"):
             encode_png(torch.empty((5, 100, 100), dtype=torch.uint8))
 
+    def test_read_1_bit_png(self):
+        try:
+            from cv2 import imread
+        except ImportError:
+            return
+
+        shapes = [
+            (27, 27),
+            (60, 60),
+            (105, 105),
+        ]
+        with get_tmp_dir() as root:
+            for shape in shapes:
```

Comment:
let's parametrize over the shapes instead of writing a for loop, as with the other tests in this file

Reply:
Makes sense, thanks!